src/main/kotlin/no/nav/syfo/persistering/PersisterSykmeldingQueries.kt

```diff
@@ -52,21 +54,24 @@ private fun Connection.updateSykmeldingsopplysninger(sykmeldingsopplysninger: Sy
             partnerreferanse = ?
         where id = ?;
     """).use {
-        it.setString(1, sykmeldingsopplysninger.pasientFnr)
-        it.setString(2, sykmeldingsopplysninger.pasientAktoerId)
-        it.setString(3, sykmeldingsopplysninger.legeFnr)
-        it.setString(4, sykmeldingsopplysninger.legeAktoerId)
-        it.setString(5, sykmeldingsopplysninger.mottakId)
-        it.setString(6, sykmeldingsopplysninger.legekontorOrgNr)
-        it.setString(7, sykmeldingsopplysninger.legekontorHerId)
-        it.setString(8, sykmeldingsopplysninger.legekontorReshId)
-        it.setString(9, sykmeldingsopplysninger.epjSystemNavn)
-        it.setString(10, sykmeldingsopplysninger.epjSystemVersjon)
-        it.setTimestamp(11, Timestamp.valueOf(sykmeldingsopplysninger.mottattTidspunkt))
-        it.setString(12, sykmeldingsopplysninger.tssid)
-        it.setObject(13, sykmeldingsopplysninger.merknader?.toPGObject())
-        it.setString(14, sykmeldingsopplysninger.partnerreferanse)
-        it.setString(15, sykmeldingsopplysninger.id)
+        var i = 1
+        it.setString(i++, sykmeldingsopplysninger.pasientFnr)
+        it.setString(i++, sykmeldingsopplysninger.pasientAktoerId)
+        it.setString(i++, sykmeldingsopplysninger.legeFnr)
+        it.setString(i++, sykmeldingsopplysninger.legeHpr)
+        it.setString(i++, sykmeldingsopplysninger.legeHelsepersonellkategori)
+        it.setString(i++, sykmeldingsopplysninger.legeAktoerId)
+        it.setString(i++, sykmeldingsopplysninger.mottakId)
+        it.setString(i++, sykmeldingsopplysninger.legekontorOrgNr)
+        it.setString(i++, sykmeldingsopplysninger.legekontorHerId)
+        it.setString(i++, sykmeldingsopplysninger.legekontorReshId)
+        it.setString(i++, sykmeldingsopplysninger.epjSystemNavn)
+        it.setString(i++, sykmeldingsopplysninger.epjSystemVersjon)
+        it.setTimestamp(i++, Timestamp.valueOf(sykmeldingsopplysninger.mottattTidspunkt))
+        it.setString(i++, sykmeldingsopplysninger.tssid)
+        it.setObject(i++, sykmeldingsopplysninger.merknader?.toPGObject())
+        it.setString(i++, sykmeldingsopplysninger.partnerreferanse)
+        it.setString(i++, sykmeldingsopplysninger.id)
```

Comment:
Lettere å legge nye ting på slutten, minimerer risikoen for å gjøre feil

Reply:
Godt tips!